docs/python-examples/fixtures/test_fixtures.py

```diff
@@ -0,0 +1,134 @@
+import logging
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+
+def alter_pvc(pvc):
+    """
+    Dummy helper function to alter pvc
```

Comment:
dont think this test is necessary due to class methods for altering within test.

Reply:
same